plugins/auth-backend/config.d.ts

```diff
@@ -67,6 +67,24 @@ export interface Config {
       };
     };
 
+    /** To control which token issuer to use, either left empty to use the
+     * default token issuer, or a complete configuration for the mounted token
+     * issuer to be used instead. */
+    issuer?: {
+      mountedTokenIssuer?: {
```

Comment:
From the users point of view I'm thinking that this is more in the direction of "key management" or something like that right? I think we could take that even a bit further and just have the config be a set of static signing keys perhaps? Initially I was thinking something like this:

```ts
auth?: {
  staticSigningKeys?: { ... }
}
```

But I realized we already have the `keyStore` config, which is the way these keys are stored. Perhaps we could piggyback on that, since we won't be using the key store with a static set of keys. Perhaps we could expand it to the following?

```ts
auth?: {
  keyStore?: {
    provider?: 'database' | 'memory' | 'firestore' | 'static';
    firestore?: {
      ...
    };
    static?: {
      keys: Array<{
        publicKeyFile: ...
        privateKeyFile?: ... // optional, must be declared at least once and the first one will be used for signing
        ...
      }>
    }
  };
};
```

As a side note I'm feeling it's important to be able to configure multiple public keys. That'll allow for seamless key rotation.

Reply:
I think I understand what you mean, I've set aside some time next Monday to address all your comments!